pkg/antctl/transform/endpoint/transform.go

```diff
@@ -0,0 +1,129 @@
+// Copyright 2020 Antrea Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package endpoint
+
+import (
+	"github.com/vmware-tanzu/antrea/pkg/antctl/transform"
+	"github.com/vmware-tanzu/antrea/pkg/apis/networking/v1beta1"
+	"github.com/vmware-tanzu/antrea/pkg/controller/networkpolicy"
+	"io"
+	"reflect"
+	"strconv"
+)
+
+type Policy struct {
+	Name string
+}
+
+type Response struct {
```

Comment:
Since the output function (`tableOutputForQueryCommands`) is specific to "query endpoint", I don't think we need to define a custom "Transform" at all. We could have `tableOutputForQueryCommands` (or `tableOutputForQueryEndpointCommand`?) operate directly on the server response struct (see a command like "trace-packet" for an example).

That would be much cleaner IMO and it would make the diff smaller as well.

Reply:
@jakesokol1 this is my most important comment I think. Are you working on addressing it?